jmx/jmx-mbeanserver-platform/pom.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.apache.aries.jmx</groupId>
+        <artifactId>org.apache.aries.jmx.parent</artifactId>
+        <version>1.1.5</version>
+        <relativePath>../jmx-parent</relativePath>
+    </parent>  
+
+    <artifactId>org.apache.aries.jmx.mbeanserver-platform</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <packaging>bundle</packaging>
+    <name>Apache Aries JMX MBeanServer (Platform)</name>
+    <description>
+        This bundle registers a MBeanServer service based on the Platform MBeanServer.
+    </description>
```

Comment:
please add `scm` tag which will be necessary for the release:

```
    <scm>
        <connection>scm:git:git@github.com:apache/aries.git</connection>
        <developerConnection>scm:git:git@github.com:apache/aries.git</developerConnection>
        <tag>HEAD</tag>
    </scm>
```

Reply:
@alien11689 added and re-pushed changes, see https://github.com/apache/aries/pull/296/files#diff-99988e5bab7ebf894524bee1b5f2bf8109fb3500902610c4bfb7580ddebad0b7 .